config.py

```diff
@@ -30,6 +30,31 @@ class Config(object):
     PASSWORD_MIN_LENGTH = 8
     PASSWORD_MAX_LENGTH = 160
 
+    CASE_URL = os.getenv('CASE_URL')
+    CASE_USERNAME = os.getenv('CASE_USERNAME')
+    CASE_PASSWORD = os.getenv('CASE_PASSWORD')
+    CASE_AUTH = (CASE_USERNAME, CASE_PASSWORD)
+
+    COLLECTION_EXERCISE_URL = os.getenv('COLLECTION_EXERCISE_URL')
+    COLLECTION_EXERCISE_USERNAME = os.getenv('COLLECTION_EXERCISE_USERNAME')
+    COLLECTION_EXERCISE_PASSWORD = os.getenv('COLLECTION_EXERCISE_PASSWORD')
+    COLLECTION_EXERCISE_AUTH = (COLLECTION_EXERCISE_USERNAME, COLLECTION_EXERCISE_PASSWORD)
+
+    IAC_URL = os.getenv('IAC_URL')
```

Comment:
These need added in docker-dev too.

Reply:
I think they should all now be listed on the confluence page.

Reply:
They still need to be added to the ras-rm-docker-dev and to the concourse pipeline before adding these changes to master.